modules/frontend/tag_sharder.go

```diff
@@ -213,7 +213,7 @@ func (s searchTagSharder) RoundTrip(pipelineRequest pipeline.Request) (pipeline.
 
 	// build request to search ingester based on query_ingesters_until config and time range
 	// pass subCtx in requests, so we can cancel and exit early
-	ingesterReq, err := s.ingesterRequest(ctx, tenantID, pipelineRequest, searchReq)
+	ingesterReq, err := s.ingesterRequest(tenantID, pipelineRequest, searchReq)
```

Comment:
this context that we are not passing anymore comes from the tracer. Maybe we should be passing it on to the   cloneChildRequest to not lose the tracing context

Reply:
the context was an unused param so i removed it. context should be preserved b/c we are using the context of the pipeline.Request when we clone